src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -189,13 +189,20 @@ private void RetrieveMatchingProcessesByProcessName()
                 if (!found &&
                     !WildcardPattern.ContainsWildcardCharacters(pattern))
                 {
+                    string val = "NoProcessFoundForGivenName";
```

Comment:
nit: `errorName` is a more descriptive variable name
```suggestion
                    string errorName = "NoProcessFoundForGivenName";
```